packages/electron-builder-http/src/publishOptions.ts

```diff
@@ -115,7 +120,8 @@ export interface S3Options extends PublishConfiguration {
 }
 
 export function s3Url(options: S3Options) {
-  let url = `https://${options.bucket}.s3.amazonaws.com`
+  let url = `https://s3.${options.region}.amazonaws.com/${options.bucket}`
+
   if (options.path != null) {
```

Comment:
As this URL format can leads to issues, let's use it only bucket name contains dots.